### ISSUE TITLE
Public : Image Slider 0x0 px

### ISSUE THREAD

**Issue by R-Men** [rohsyl/omega-l#16]
When creating an image slider the images won't show in public side. 

I had to add in my css these lines : 

```
.om-wrapper{
    min-width: xxxpx;
    margin: 0 auto;
}
```

**Comment by rohsyl** [comment]
I will add a default css rules for this.
And the rule can be overloaded in the theme css if needed...

**Comment by R-Men** [comment]
mmh adding css doesn't work.. I still have problems showing slider

**Comment by R-Men** [comment]
Okay I found the problem.. It seems that jquery is undefined in the public part.. 
You should check that @rohsyl

**Comment by rohsyl** [comment]
A fast fix would be to add jquery in the `<head>` of your theme

**Comment by R-Men** [comment]
Yeah thx that could do the trick but a real fix for it would be great
